src/pages/dashboard.vue

```diff
@@ -40,22 +47,52 @@ const showCustodian = computed(() => {
   return hasCustodian.value && currentBusiness.value.state === BusinessStateE.HISTORICAL
 })
 
+const bootstrapOfficeTitle = computed(() => {
+  return bootstrapFilingType.value === FilingTypes.REGISTRATION
+    ? t('title.section.businessAddresses')
+    : t('title.section.officeAddresses')
+})
+const bootstrapOffices = computed(() => {
+  return bootstrapFilingType.value === FilingTypes.REGISTRATION
+    ? [{}]
+    : [{ title: t('label.address.officeType.registered') }, { title: t('label.address.officeType.records') }]
+})
+
+const bootstrapPartiesTitle = computed(() => {
+  if (bootstrapFilingType.value === FilingTypes.REGISTRATION) {
+    return bootstrapLegalType.value === CorpTypeCd.SOLE_PROP
+      ? t('title.section.proprietors')
+      : t('title.section.partners')
+  }
+  return t('title.section.currentDirectors')
+})
+
 // check if the business has a party that has a certain role type
 const containRole = (roleType) => {
-  return business.currentParties.parties.find(party =>
+  return currentParties.value?.parties.find(party =>
     party.roles.find(role => role.roleType === roleType && !role.cessationDate)
   )
 }
-const loadBusinessInfo = () => {
-  if (route.params.identifier) {
-    business.loadBusinessAddresses(route.params.identifier as string)
-    business.loadParties(route.params.identifier as string)
-    business.loadBusiness(route.params.identifier as string)
+const loadBusinessInfo = async () => {
+  const identifier = route.params.identifier as string
+  if (identifier) {
+    if (bootstrap.checkIsTempReg(identifier)) {
+      // this is a business bootstrap (actual business does not exist yet)
+      await bootstrap.loadBusinessBootstrap(identifier)
+      useBcrosTodos().loadBootstrapTask({ enabled: true, order: 0, task: bootstrapFiling.value } as TaskI)
+    } else {
+      await business.loadBusiness(identifier)
+      business.loadBusinessAddresses(identifier)
```

Comment:
do other need await too ?
or we just need to await on business stuff, other can be done in any order/parallel ?

Reply:
exactly - once the business is loaded the other stuff can happen in parallel